lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1299,13 +1304,20 @@ void MtohRenderOverride::_CreateSceneIndicesChainAfterMergingSceneIndex(const MH
         _sceneIndexRegistry.reset(new MayaHydraSceneIndexRegistry(_renderIndexProxy));
     }
 
+    const SdfPath mayaNativeRoot("/MayaHydraViewportRenderer");
+
+    // Add display style scene index
+    _lastFilteringSceneIndexBeforeCustomFiltering = _displayStyleSceneIndex =
+            Fvp::DisplayStyleOverrideSceneIndex::New(_lastFilteringSceneIndexBeforeCustomFiltering);
+    _displayStyleSceneIndex->addExcludedSceneRoot(mayaNativeRoot); // Maya native prims don't use global refinement
+
     auto wfSi = TfDynamic_cast<Fvp::WireframeSelectionHighlightSceneIndexRefPtr>(Fvp::WireframeSelectionHighlightSceneIndex::New(_lastFilteringSceneIndexBeforeCustomFiltering, _selection));
     wfSi->SetDisplayName("Flow Viewport Wireframe Selection Highlight Scene Index");
     
     // At time of writing, wireframe selection highlighting of Maya native data
     // is done by Maya at render item creation time, so avoid double wireframe
     // selection highlighting.
-    wfSi->addExcludedSceneRoot(_ID);
```

Comment:
@ppt-adsk while debugging around I found that the call that you implemented to wfSi->addExcludedSceneRoot doesn't receive the proper path. In my case the exclude path was "/MayaHydraViewportRenderer/_MayaHydra_HdStormRendererPlugin_0000023A87F10B80" while all the Maya native primitives were at the path "/MayaHydraViewportRenderer/rprims" so essentially we were drawing the selection highlight two times for those. Here I fix the bug

Reply:
@vlasovi thanks for the fix!  Can we better document what _ID is supposed to be used for?  Also, `/MayaHydraViewportRenderer` is duplicated at line 566.  Can we factor this out?